organizational-structure-overview.md

```diff
@@ -0,0 +1,87 @@
+# OpenSSF Governance & Org Structure
+
+## OpenSSF Charter
+
+The overall governance of the OpenSSF is defined in [the Foundation Charter](https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf) (PDF file). 
+
+
+## Definitions
+
+The OpenSSF is comprised of instances of the following categories of official groups:
+
+- The **Governing Board** (GB) oversees the budget and legal status/structure of the OpenSSF. Board meetings may or may not be open to the public.
+- **Committees** are formed by, and report to, the Board. Broadly speaking, Committees handle non-technical matters as needed by the GB, wherein a named set of people are tasked with handling a specific objective. Like the Board meetings, Committee meetings may or may not be open to the public.
+- The **Technical Advisory Council** (TAC) is responsible for the general success of all Technical Initiatives (defined below). TAC meetings are generally open to the public, with the exception of special sessions. 
+  - TAC membership is limited to seven (7) seats, with alternating elections every year. Contributors to Technical Initiatves may run, and each contributor may vote once in TAC elections. 
+- **Technical Initiatives** are groups formed in support of the overall mission of the OpenSSF. Technical Initiatives include Working Groups (WG), Special Interest Groups (SIG), Projects, and services governed by their own Charters with various objectives, reporting relationships, and funding models described below.
+   - All Technical Initiatives are open groups focused on a technical objective, with transparent proceedings that are open to the public, and vary in their objectives.
+   - Technical Initiatives usually manage one or more GitHub repos of their own, may have separate meetings.
+- A **Project** is a Technical Initiative focused on the development and ongoing support of open source licensed software (source code) and its supporting artifacts (technical documentation, etc.).
+- A **Working Group** (WG) focuses on developing other types of deliverables than software such as guides, specifications, and educational material, or conducting initiatives such as an education outreach. While a WG does not produce open source software as a primary artifact it may launch Projects that do so. A WG may also launch Special Interest Groups (SIG) to perform specific tasks. WGs often include some open source code, or use licensed software, in fulfilment of their Charter.
```

Comment:
I feel it's cleaner, and easier to explain, to stick to how it was explained to me when I arrived, that "Working Groups are the primary organizational/delegation structure in OpenSSF" - meaning everything interesting happens within those WGs, and the TAC is there to make sure the trains run on time and generally speaking to the right destination. I think it gets complicated if we have SIGs and Projects that might report directly to the TAC, plus it helps with thematic alignment and converging of efforts if all, say, "digital signing" related Projects and SIGs were in the same WG.

Reply:
I agree with this. The TAC already is stretched pretty thin, adding SIGs and projects will create a lot of work

Reply:
+1